zabbix/map.jinja

```diff
@@ -149,6 +149,20 @@
     },
   },
 
+  'Windows':{
+    'user': 'Administrator',
+    'group': 'Administrators',
+    'agent': {
+      'version': '3.0.28.2400',
+      'pkgs': ['zabbix-agent'],
+      'service': 'Zabbix Agent',
+      'config': 'C:\\Program Files\\Zabbix Agent\\zabbix_agentd.conf',
+      'logfile': 'C:\\Program Files\\Zabbix Agent\\zabbix_agentd.log',
```

Comment:
Set some sane default value for `logfilesize` to prevent uncontrollable grow of log file.
```suggestion
      'logfile': 'C:\\Program Files\\Zabbix Agent\\zabbix_agentd.log',
      'logfilesize': '5',
```